examples/minimal/public/robots.txt

```diff
@@ -1,2 +0,0 @@
-User-agent: *
-Disallow: /
```

Comment:
why delete robots.txt?

Reply:
None of the other examples ships it and no one in the discussion on Discord seemed to know its purpose 😅

I think that while it's a fun way of showing off that, yes you should put your robots.txt here, shipping one that disable robots crawling your website might lead to people shipping it by accident in their projects (especially considering this example is supposed to be "minimal"

Reply:
I agree about removing this for minimal. It’s more _opinionated_ than _minimal_.